lib/modules/filteReddit.js

```diff
@@ -769,21 +760,16 @@ modules['filteReddit'] = {
 			(RESUtils.currentSubreddit('all')) || (RESUtils.currentDomain()) || (RESUtils.currentMultireddit('me/f/all')),
 			onSavedPage = modules['filteReddit'].excludeSaved.test && modules['filteReddit'].excludeSaved.test(location.href);
 
-		Array.prototype.slice.call(entries).forEach(function(entry) {
-			var postSubreddit, currSub;
+		Array.prototype.slice.call(things).forEach(function(thing) {
```

Comment:
I changed `RESUtils.things` to return an array of plain objects and added `RESUtils.$things` to return a jQuery object.  So, I should just be able to call `.forEach` :+1: